source/usage-examples/updateOne.txt

```diff
@@ -3,3 +3,68 @@ Update a Document
 =================
 
 .. default-domain:: mongodb
+
+You can update a document in a collection by using the ``Collection.UpdateOne()``
+method. 
+
+This example passes a query filter and an update parameter to the
+``UpdateOne()`` method. The query filter specifies the document to
+match, while the update parameter specifies the change to
+make to the document. The following code example calls the
+``UpdateOne()`` method on a document matched by its ``_id`` field from
+the ``restaurants`` collection to create a new field called
+``avg_rating`` with a value of 4.4:
+
+.. include:: /includes/run-example-tip.rst
+
+.. literalinclude:: /code-snippets/updateOne.go
+   :start-after: begin updateone
+   :end-before: end updateone
+   :emphasize-lines: 6
+   :language: go
+   :dedent:
+
+Click here <TODO: github link to file> to see a fully runnable example.
+
+.. note::
+
+   If the query filter does not match any documents, the operation will
+   return a result with a ``MatchedCount`` property of 0. If the filter
+   matches multiple documents, the method will select and update one document.
+
+Expected Result
+---------------
+
+If you execute a :doc:`find
+</fundamentals/crud/read-operations/retrieve/#find>` operation querying
+for the same ``_id`` value used in the example, you should see that the
+matched document now includes the ``avg_rating`` field:
+
+.. code-block:: json
+   :copyable: false
+
+   // result truncated
+   { 
+      "_id" : ObjectId("5eb3d668b31de5d588f42a7a"),
+      ...
+      "name" : "Green House Cafe",
+      "restaurant_id" : "40372112",
+      "avg_rating" : 4.4
+   }
+
+Additional Information
+----------------------
+
+For more information on updating documents, specifying query filters, and
+handling potential errors, see our guide on :doc:`changing a document
+</fundamentals/crud/write-operations/change-a-document/>`.
+
+For more information on update operators,
+see the :manual:`MongoDB update operator reference documentation
+</reference/operator/update/#update-operators>`.
```

Comment:
Same as the previous suggestion, let's clearly mark links that don't work as TODO

Reply:
done